applications/RomApplication/python_scripts/calculate_rom_basis_output_process.py

```diff
@@ -136,6 +135,9 @@ def ExecuteFinalize(self):
 
         # Set a NumPy array with the snapshots data
         n_nodes = self.model_part.NumberOfNodes()
+        n_elements = self.model_part.NumberOfElements()
+        rom_basis_dict["hrom_settings"]["original_number_of_elements"] = n_elements
+        rom_basis_dict["hrom_settings"]["hrom_format"] = self.rom_basis_output_format
```

Comment:
```suggestion
        rom_basis_dict["hrom_settings"]["output_format"] = self.rom_basis_output_format
```
I think `hrom` is redundant in this case. Besides, I also think that `output_format` is more self-descriptive.